docs/hugo/content/guide/crd-management.md

```diff
@@ -163,3 +163,7 @@ that removed those CRDs. **Note:** This is not yet implemented, it is being trac
   `dbformysql.azure.com/FlexibleServer` may be specified as well, but there are often other resources in the 
   group which pair together to enable other scenarios, such as `dbformysql.azure.com/FlexibleServersFirewallRules`.
   It's generally easier to just include the whole group.
+
+## Entra Resources
```

Comment:
Wondering if it makes more sense to put this into the `authentication` sections rather than CRD management?

This section is more about how ASO manages CRDs and less about configuring the secret, whereas I feel like the auth sections are a lot more about the secret.